powerline/renderers/lemonbar.py

```diff
@@ -36,10 +37,26 @@ def hl(self, contents, fg=None, bg=None, attrs=None):
 		return text + contents + '%{F-B--u}'
 
 	def render(self, *args, **kwargs):
+		self.segment_info['output'] = kwargs.get('matcher_info')
```

Comment:
shouldn't I merge `self.segment_info` or copy it?

    segment_info = self.segment_info.copy()
    segment_info['output'] = kwargs.get('matcher_info')
    ....